### PR TITLE
ledger 3.1.1

### PR DESCRIPTION
**Pull request by afh**
- addresses build issues with newer boost versions >=1.58.0
  - remove patches merged upstream
  - remove utfcpp resource (now included with the source)

Please note #47125 is waiting on these changes.

### CODE REVIEW

File: Library/Formula/ledger.rb [hunk 1]
```diff
@@ -96,22 +65,7 @@ def install
     assert_equal 0, $?.exitstatus
 
     if build.with? "python"
-      system "python", "#{share}/ledger/demo.py"
+      system "python", share/"ledger/demo.py"
```
Comment (bfontaine) [inline Library/Formula/ledger.rb]: You can use `#{pkgshare}/demo.py` here.

Comment (afh) [inline Library/Formula/ledger.rb]: Thanks @bfontaine, I've updated the pull request accordingly with consistent use of `pkgshare` over `share+"ledger"`.

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Comment by afh** [comment]
Thanks for merging!

**Pull request closed without merging**